admin/plugins.php

```diff
@@ -76,7 +75,7 @@ function perflab_render_plugins_ui() {
 		return;
 	}
 	?>
-	<div class="wrap">
+	<div class="wrap plugin-install-php">
```

Comment:
@mukeshpanchal27 What is the purpose of this class?

Reply:
The `.plugin-install-php` selector is targeted by some of the WP admin styles.

Reply:
But is that the right class to use here? I don't see any application of the class in core when searching the codebase, which makes me wonder if this is the legacy class. Maybe I'm missing something. But if core doesn't use it, then why should we here?

Reply:
Yes. This class name _is_ used by core. This is needed here so that all the plugin cards have the equal height. This is because the core CSS targets `.plugin-install-php .plugin-card` to make the cards use flexbox

Reply:
But where is it used in core? I only found it in CSS and JS but nowhere actually added in the HTML markup.

Reply:
Go to `wp-admin/plugin-install.php` and you'll see it on the `<body>` element. It's a dynamic class based on the file name (`plugin-install.php` -> `plugin-install-php`)

Reply:
Ah I see, thanks for clarifying!

Reply:
Thank you, @swissspidy, for responding to Felix's questions.